docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "3.8"
+services:
+  db:
+    image: mysql:8.0
+    restart: unless-stopped
+    env_file:
+      - "./.env"
+    environment:
+      MYSQL_ROOT_PASSWORD: $MYSQL_ROOT_PASSWORD
+      MYSQL_DATABASE: $MYSQL_DATABASE
+      MYSQL_USER: $MYSQL_USER
+      MYSQL_PASSWORD: $MYSQL_PASSWORD
```

Comment:
correct syntax
![image](https://github.com/yuriy-kononchuk/Online-Book-store-App-YK/assets/49767049/06f8dbd6-28a0-4d12-b6f6-6a165ee0b8fd)

root DB and password should be enough